examples/monitoring/openai_proxy_quickstart_enterprise_mode.ipynb

```diff
@@ -0,0 +1,280 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 1: Configure target URL\n",
+    "WANDB_BASE_URL = \"https://api.wandb.ai\" # CHANGE ME!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 2: Ensure WANDB_API_KEY is set in env\n",
+    "import os\n",
+    "from getpass import getpass\n",
+    "\n",
+    "WANDB_APP_URL = WANDB_BASE_URL if WANDB_BASE_URL != \"https://api.wandb.ai\" else \"https://app.wandb.ai\"\n",
+    "WEAVE_ROOT_URL = WANDB_BASE_URL + \"/weave\" if WANDB_BASE_URL != \"https://api.wandb.ai\" else \"https://weave.wandb.ai\"\n",
+    "if os.getenv(\"WANDB_API_KEY\") is None:\n",
+    "    os.environ[\"WANDB_API_KEY\"] = getpass(f\"Paste your OpenAI key from: {WANDB_APP_URL}/authorize \\n\")\n",
+    "print(\"W&B API key configured\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 3: Perform setup\n",
+    "import openai\n",
+    "openai.api_base = f\"{WANDB_BASE_URL}/proxy/openai/v1\"\n",
+    "openai.api_key = os.getenv(\"WANDB_API_KEY\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Start logging\n",
+    "You can use your existing API calls as-is, now with extra logging!\n",
+    "\n",
+    "By default, this will log to `{enterprise_admin_entity}/monitoring/openai`, where the project is `monitoring` and the table name is `openai`.\n",
+    "\n",
+    "### Advanced configuration\n",
+    "1. Optionally add custom attributes to your table with the `X-Wandb-Attribute-` prefix.  For example, adding the header `X-Wandb-Attribute-Foo: bar` will add a column `Foo` with the value `bar` to your table.\n",
+    "2. Optionally add the `X-Wandb-Client-Id` header if you want to further group related requests together.  By default, each request will have its own unique Client ID.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Style 1: Using OpenAI SDK"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = openai.ChatCompletion.create(\n",
+    "    model=\"gpt-3.5-turbo\",\n",
+    "    messages=[{\"role\": \"user\", \"content\": \"Tell me a joke about gradients as if you were Shakespeare\"}],\n",
+    "    headers={\n",
+    "        # Optional Client ID\n",
+    "        \"X-Wandb-Client-Id\": \"Shakespearean-Gradients-001\",\n",
+    "        # Optional Key-Value pairs:\n",
+    "        \"X-Wandb-Attribute-Persona\": \"Shakespeare\",\n",
+    "    }\n",
+    ")\n",
+    "response"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Admins can view logs at {WEAVE_ROOT_URL}/browse/wandb/[ADMIN_ENTITY]/monitoring/openai\")"
```

Comment:
I really like what you're doing with ADMIN_ENTITY, and maybe it's better to tell folks to optionally set it at the top? then these urls would potentially work if they know the admin entity for their instance!